### ISSUE TITLE
Bundle contains many lodash functions

### ISSUE THREAD

**Issue by krystian50** [lodash/babel-plugin-lodash#236]
## Why the build size is so large? 
Working example: https://github.com/krystian50/babel-plugin-lodash-issue
The only lodash related code is:
```
import './index.html';
import _fp from 'lodash/fp';

const value = 
    _fp.map(el => el * 3)
([1, 2])

console.log(value);
```
Plugin is added through `.babelrc`

Without babel-plugin-lodash build looks like:
![Screenshot 2020-04-16 at 12 39 27](https://user-images.githubusercontent.com/19366440/79446922-593fa800-7fdf-11ea-895c-6fb53941e275.png)

With plugin:
![Screenshot 2020-04-16 at 12 39 56](https://user-images.githubusercontent.com/19366440/79446968-69f01e00-7fdf-11ea-9e65-4aa2b4c71f69.png)

Basically there is many functions, but I only used map. I checked build code and there is import lines:
```
"use strict";
__webpack_require__.r(__webpack_exports__);
/* harmony import */ var lodash_fp_map__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! lodash/fp/map */ "./node_modules/lodash/fp/map.js");
/* harmony import */ var lodash_fp_map__WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/__webpack_require__.n(lodash_fp_map__WEBPACK_IMPORTED_MODULE_0__);
/* harmony import */ var _index_html__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./index.html */ "./src/index.html");

```

**Comment by hingermayank** [comment]
@krystian50 hi, any luck? 

Happening the same for me. I just used one function but the bundle had too many functions.

<img width="1225" alt="Screenshot 2020-04-23 at 4 15 51 AM" src="https://user-images.githubusercontent.com/6342096/80041528-4fa3cc00-851a-11ea-8ae3-9f481a91a8ae.png">

**Comment by johnny-b-good** [comment]
We use Semantic UI React that depends on this plugin to strip unnecessary Lodash methods. As a result, the bundle became much larger than it could be.

**Comment by evenfrost** [comment]
Any update?

**Comment by evenfrost** [comment]
@jdalton is it still working?

**Comment by grzegorzjudas** [comment]
It's still happening. I've prepared a repro repo: [grzegorzjudas/babel-lodash-repro](https://github.com/grzegorzjudas/babel-lodash-repro).

**Comment by 0xdevalias** [comment]
Potentially related:

- https://github.com/lodash/babel-plugin-lodash/issues/243
